src/main2.rs

```diff
@@ -0,0 +1,53 @@
+use cpal::{traits::{DeviceTrait, HostTrait, StreamTrait}, StreamConfig};
+use rustfft::{num_complex::Complex, FftPlanner};
+
+fn main() {
+    let stream = build_stream();
+    loop {
+    stream.play().expect("stream couldn't play?");
+    }
+}
+
+fn build_stream() -> cpal::Stream {
```

Comment:
[FYI] In a console application or example, using `.expect()` to handle errors is totally fine. In a GUI application, though, you wouldn't want to use `.expect()` in a function like this because the app could crash if it fails. Instead, you can use `Result` and [error propagation](https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html#propagating-errors) to gracefully handle failures.

Reply:
Totally, I am planning on making this a GUI application so planning on working on error propagation.